### PR TITLE
update instructions for CBUI changes - merge Dec or Jan

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 6354

Apply CBUI changes to the instructions and update the language around account types.

### CODE REVIEW

File: content/how-to/account-management/combine-invoices-for-rackspace-managed-and-cloud-accounts/index.md [hunk 1]
```diff
@@ -16,35 +16,38 @@ combine your bills into a single invoice and pay only once. This article describ
 how to combine invoices, how your billing due date is affected, and how to
 separate a combined invoice.
 
-**Note:** Currently, you can combine bills from only Rackspace Managed and
-Rackspace Cloud accounts. You cannot combine accounts that are billed in different
-currencies, are already combined with another invoice, are contracted with different
+**Note:** Currently, you can combine bills from Rackspace Private Cloud,
+Rackspace Public Cloud, and third-party cloud accounts, such as Amazon Web
+Services&reg; (AWS), Microsoft&reg; Azure&trade;, and Google Cloud
```
Comment (robb-romans) [inline content/how-to/account-management/combine-invoices-for-rackspace-managed-and-cloud-accounts/index.md:21]: For clarity, suggest converting these sentences to unordered lists.

File: content/how-to/account-management/combine-invoices-for-rackspace-managed-and-cloud-accounts/index.md [hunk 2]
```diff
@@ -53,23 +56,23 @@ After the accounts are consolidated, the **Invoiced Accounts** section of the
...
-is billed on the first day of the month.
+If your combined invoice includes a Rackspace Private Cloud or third-party
+cloud account, your next invoice is billed on the first day of the month.
 
 The following example describes how your billing is affected when you combine
-Cloud and Managed account invoices:
+Rackspace Public Cloud and AWS account invoices:
 
-- The Managed account is the primary account with a billing date of February 1, and the Cloud account's billing date is February 15.
+- The AWS account is the primary account with a billing date of February 1, and the Public Cloud account's billing date is February 15.
```
Comment (robb-romans) [inline content/how-to/account-management/combine-invoices-for-rackspace-managed-and-cloud-accounts/index.md:68]: remove possessive. "the billing date for the Public Cloud account is.."

**Comment by stephamon** [comment]
The billing UI changes are due to go live in December or early January, so this can be merged after that happens.

**Review by cal27279 — APPROVED** [review]
lgtm

**Pull request merged**